apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -398,9 +419,16 @@ def from_h5ad(
 
           .. code-block:: python
 
-              with soma.open(uri, 'w') as exp:
-                  exp.metadata.update({"aaa": "BBB"})
-                  exp.obs.metadata.update({"ccc": 123})
+              with soma.open(
+                  uri, "w"
+              ) as exp:
+                  exp.metadata.update({
+                      "aaa": "BBB"
+                  })
+                  exp.obs.metadata.update({
+                      "ccc": 123
+                  })
```

Comment:
I think the old formatting was better. Was this an intentional change or something automated? (Similarly with the other formatting-only changes).

Reply:
Whoops, I was experimenting with ruff's docstrings formatter and didn't notice it made these changes. Will revert.